RecoBTag/MXNet/python/Parameters/ParticleNet/V00/pfMassDecorrelatedParticleNetPreprocessParams_cfi.py

```diff
@@ -0,0 +1,246 @@
+import FWCore.ParameterSet.Config as cms
+
+pfMassDecorrelatedParticleNetPreprocessParams = cms.PSet(
+    input_names = cms.vstring(
+        'pf_points',
+        'pf_features',
+        'pf_mask',
+        'sv_points',
+        'sv_features',
+        'sv_mask'
+    ),
+    pf_features = cms.PSet(
+        input_shape = cms.vuint32(1, 20, 100),
+        var_infos = cms.PSet(
+            pfcand_VTX_ass = cms.PSet(
+                median = cms.double(7.0),
+                norm_factor = cms.double(1.0)
```

Comment:
> are these rescaling parameters specific to the trained model?
> Wouldn't it be better then to distribute these constants in the data files so that they more coherently stay with the model distribution.

This would be a better way I think. I will do it in a separate PR.
Which (structured) format is the easiest to parse in CMSSW? JSON (which JSON parse do we use)?